### PR TITLE
Add 2 new Challenges Addon placeholders

### PR DESCRIPTION
**Pull request by BONNe**

### CODE REVIEW

File: placeholders.csv [hunk 1]
```diff
@@ -42,15 +42,17 @@ BentoBox,[gamemode]_resets,Number of times the player has reset his island,1.5.0
...
-Challenges,[gamemode]_challenges_latest_level_id,Latest unlocked challenge level id,0.8.2
+Challenges,[gamemode]_challenges_total_completion_count,Number of times the player completed challenges,0.9.0
+Challenges,[gamemode]_challenges_completed_count,Number of challenges the player has completed,0.9.0
+Challenges,[gamemode]_challenges_uncompleted_count,Number of challenges the player has not yet completed,0.9.0
+Challenges,[gamemode]_challenges_completed_level_count,Number of levels the player has completed,0.9.0
+Challenges,[gamemode]_challenges_uncompleted_level_count,Number of levels the player has not yet completed,0.9.0
+Challenges,[gamemode]_challenges_unlocked_level_count,Number of levels that are unlocked for the player,0.9.0
+Challenges,[gamemode]_challenges_locked_level_count,Number of levels that are locked for the player,0.9.0
+Challenges,[gamemode]_challenges_latest_level_name,Latest unlocked challenge level name,0.9.0
+Challenges,[gamemode]_challenges_latest_level_id,Latest unlocked challenge level id,0.9.0
```
Comment (Poslovitch) [inline placeholders.csv:53]: Why did you change the versions for those?

Comment (BONNe) [reply placeholders.csv:53]: Well, in 0.8.2 they are `challenges_[gamemode]_[placeholder]`
Only in 0.9.0 they are in correct format.

Also, there is no @deprecated tag to add, for placeholders that will be removed later...

Comment (Poslovitch) [reply placeholders.csv:53]: > Well, in 0.8.2 they are challenges_[gamemode]_[placeholder]
> Only in 0.9.0 they are in correct format.

Alright, makes sense.

> Also, there is no @ deprecated tag to add, for placeholders that will be removed later...

You mean in the docs or in the code? Because I don't think we can do that in the code ahah.

Comment (BONNe) [reply placeholders.csv:53]: In docs.

Comment (Poslovitch) [reply placeholders.csv:53]: Oh, I could that quite easily.
For now I'll just merge the PR as is :)

**Comment by Poslovitch** [comment]
Aside from a "feature" request to be able to tag placeholders as "deprecated", I'll merge this PR as is. Thanks @BONNe !
